pyndl/preprocess.py

```diff
@@ -141,8 +142,9 @@ def process_occurrences(occurrences, outfile, *,
 
 def create_event_file(corpus_file,
                       event_file,
-                      symbols="abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ",
```

Comment:
If we want to remove symbols, we should include a fallback valid_symbols=symbols that posts a deprecation warning to avoid breaking other peoples code.